pkgs/development/python-modules/geopandas/default.nix

```
@@ -14,7 +14,7 @@
 
 buildPythonPackage rec {
   pname = "geopandas";
-  version = "0.14.3";
+  version = "0.14.4";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
Could you please use a new parameter, `pyproject` instead to prevent legacy build?
Also, we need to add `setuptools` to `build-system`. Thanks.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters

---

Yes, we can update that.

---

Done in dd03cbc938f91f500c14241bb065da31df8858a6 .